.gitlab-ci.yml

```diff
@@ -141,6 +141,8 @@ build-android:
 build-linux:
   <<:                              *build-on-linux
   only:                            *releaseable_branches
+  after_script:
+    - cp -r scripts/docker/hub/* ./artifacts/
```

Comment:
I'm a bit worried if `Dockerfile` and `check-sync.sh` will be continuously published from now on

Reply:
files can be synced explicitly for publishing

Reply:
what do you mean?

Reply:
I just wanted to mention that if you don't want to publish them (doesn't make so much sense), you can specify the files explicitly that should be published i.e. either remove them from the artifacts directory later before syncing to s3 or copying only files to s3 that are needed.

Reply:
indeed, will do it, thx